handbook/engineering/search/index.md

```diff
@@ -1,3 +1,29 @@
-# Search
+# Search team
 
-TODO: identify an owner to fill out this page and update CODEOWNERS
\ No newline at end of file
+## Vision
+
+Build and deliver Universal Code Search to all developers.
```

Comment:
I don't think we came up with a vision yet, but this seems generally true. Please suggest any better ideas.

Reply:
This is a good start and worth working on, but I wouldn't block merging this PR on improving this. Perhaps a good discussion topic for next team sync?